docs/pages/database-access/reference/configuration.mdx

```diff
@@ -51,6 +54,34 @@ proxy_service:
   mongo_public_addr: "mongo.teleport.example.com:443"
 ```
 
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+Teleport Cloud automatically configures the Teleport Proxy Service with the
+following settings that are relevant to Database Access. This reference
+configuration uses `mytenant.teleport.sh` in place of your Teleport Cloud tenant
+address.
+
+```yaml
+proxy_service:
+  enabled: "yes"
+  # PostgreSQL proxy is listening on the regular web proxy port.
+  web_listen_addr: "0.0.0.0:3080"
+  # MySQL proxy is listening on a separate port.
+  mysql_listen_addr: "0.0.0.0:3036"
+  # Database clients will connect to the Proxy Service over this hostname.
+  public_addr: "mytenant.teleport.sh:443"
+  # Address advertised to MySQL clients.
+  mysql_public_addr: "mytenant.teleport.sh:3036"
+  # Address advertised to PostgreSQL clients.
+  postgres_public_addr: "mytenant.teleport.sh:443"
+  # Address advertised to Mongo clients. If not set, public_addr is used.
+  mongo_public_addr: "mytenant.teleport.sh:443"
```

Comment:
Are there additional ports configured if Microsoft SQL Server or Redis is used?

If only one or two database types are used, are all the ports for all the database types still configured on the Proxy?

Reply:
This is based on the current Proxy Service configuration that's hardcoded for Cloud tenants. It doesn't look like there are Proxy Service settings for SQL Server and Redis.